src/Microsoft.DotNet.Build.Tasks/PackageFiles/ReferenceAssemblies.targets

```diff
@@ -24,6 +24,18 @@
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsReferenceAssembly)'=='true' AND '$(UsingMicrosoftNETSdk)'=='true'">
+    <!-- When not using Sdk-style projects, Microsoft.Common.CurrentVersion.targets,  
```

Comment:
Hopefully no hasn't already used these property so resetting them will work. Do you think it is possible to move these in the ReferenceAssembly.props instead which should be before the Microsoft.Common.CurrentVersion.targets?

Reply:
I tried moving them to the `.props` file, but I couldn't because `$(AssemblyVersion)` isn't set by that time.

Reply:
:( OK